src/rajAJun20/SumOfNumbersInStringAscii.java

```diff
@@ -0,0 +1,18 @@
+package rajAJun20;
+
+public class SumOfNumbersInStringAscii {
+	
+	void findSumOfNumbersInString(String input){
+		int sum = 0;
+		for(int index = 0; index < input.length(); index++){
+			if(input.charAt(index) >= '0' && input.charAt(index) <= '9'){
+				sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
sum += (str.charAt(index) - '0'); You can use this as well. 
Or  sum = sum + Character.getNumericValue(str.charAt(index);